tests/api_tests.js

```diff
@@ -322,8 +323,13 @@ function runNodeEnvTests(nodeEnvData, t) {
                                          t.ok(parseInt(nodeEnvData['max.old.space.size']) > 0,
                                               "max.old.space.size is positive");
 
-                                              var nodeVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
-                                              if(nodeVersion >= 6.5) { // issue 283
+                                              // converts e.g. 'v8.0.0' to [8,0.0]
```

Comment:
What's the advantage to using semver (to outweigh the downside of having another devDependency)? This should work for all node versions, and it's < 10 lines of code.